robottelo/constants.py

```diff
@@ -344,6 +345,17 @@
         'distro': DISTRO_RHEL7,
         'key': 'rhst',
     },
+    'rhst7_64': {
```

Comment:
We need change this every time then? Can we build some logic if sat_version to upgrade is n, then pick n-1

Reply:
Your point is valid however that true for upgrade ...but I am thinking if someone from tier automation wants to use it then may be a bit false..
I am not sure but we are using the constant in same way for other repos too?

Reply:
Also this change is for z stream branch hence the constant  is not much gonna to use by another tests as well.
we may think for same in master branch further ..